tests/library/browsercontext-fetch-happy-eyeballs.spec.ts

```diff
@@ -43,21 +43,24 @@ it.beforeEach(({ server }) => {
   });
 });
 
-it('get should work', async ({ context, server }) => {
+it('get should work', async ({ context, server, channel }) => {
+  it.skip(channel === 'webkit-wsl', 'server.PREFIX is the WSL->Windows address');
```

Comment:
Let's use the new `loopback` or something? We should make sure we run these tests.